neo/Prompt/InputParser.py

```diff
@@ -7,12 +7,25 @@ class InputParser:
     def parse_input(self, user_input):
         if len(user_input):
             command_parts = self.parser.parseString(user_input)
-            for n, i in enumerate(command_parts):
-                for x in i:
-                    if x != ']':
-                        break
-                    command_parts[n - 1] = command_parts[n - 1] + i
-                    command_parts.pop(n)
-                    break
+            command_parts = merge_items(command_parts)
             return command_parts[0], command_parts[1:]
         return None, None
+
+
+def merge_items(command_parts):
+    s = 0
+    f = 0
+    for n, i in enumerate(command_parts):
```

Comment:
from a codestyle perspective I found it hard to read `n` as index and `i` as element. It is usually `i` as index, and then something for element.
Can you swap them around ? because I was hurting my head while trying to follow the code because of the order. 

Also please use a bit longer names for `s` and `f`, like `start`/`finish`. They make it easier to read. thanks

Reply:
@ixje thank you for the recommendations. I will remember them in future PRs.